### PR TITLE
Added docs for go live checklist

### PR DESCRIPTION
**Pull request by martinmitrevski**
### 🎯 Goal

Provide a checklist for customers to go through before going live.

### 🛠 Implementation

Added a new doc in docusaurus.

### 🎨 Showcase

_Add relevant screenshots and/or videos/gifs to easily see what this PR changes, if applicable._

| Before | After |
| ------ | ----- |
|  img   |  img  |

### 🧪 Manual Testing Notes

_Explain how this change can be tested manually, if applicable._

### ☑️ Contributor Checklist

- [ ] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [ ] This change follows zero ⚠️ policy (required)
- [x] This change should be manually QAed
- [ ] Changelog is updated with client-facing changes
- [ ] New code is covered by unit tests
- [ ] Comparison screenshots added for visual changes
- [ ] Affected documentation updated (docusaurus, tutorial, CMS)

### CODE REVIEW

File: docusaurus/docs/iOS/guides/go-live-checklist.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+Therefore, it's important to do any other action after the `logout` has completed, as described [here](https://getstream.io/chat/docs/sdk/ios/uikit/getting-started/#disconnect--logout).
+
+
+### Controllers in SwiftUI
+
+Avoid creating controllers as computed variables in your SwiftUI code. 
+
+If you do that, every time there's a redraw, a new instance of the controller would be created, leading to unpredictable state.
+
+Better approach would be to define the controller as a `@State` or create it somewhere else and pass it to the view.
```
Comment (ipavlidakis) [inline docusaurus/docs/iOS/guides/go-live-checklist.md:58]: Isn't it `@StateObject` as controllers are classes?

File: docusaurus/docs/iOS/guides/go-live-checklist.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+To quickly get started with the chat SDK, you can use [development tokens](https://getstream.io/chat/docs/ios-swift/tokens_and_authentication/?language=swift#developer-tokens). 
+
+Development tokens disable token authentication and use client-side generated tokens or a manually generated static token. 
+
+They are not suitable for production usage. Therefore, it's important that you use a proper authentication mechanism before you release your app.
+
+For additional security, we recommend using tokens with expiry date, and our [token provider](https://getstream.io/chat/docs/ios-swift/tokens_and_authentication/?language=swift#how-to-refresh-expired-tokens) mechanism for refreshing tokens.
+
+### Don't Store Sensitive Data
```
Comment (ipavlidakis) [inline docusaurus/docs/iOS/guides/go-live-checklist.md:37]:
```suggestion
### Sensitive Data Storage
```
Don't sounds a bit aggressive

File: docusaurus/docs/iOS/guides/go-live-checklist.md (hunk 1 above)
Comment (martinmitrevski) [reply docusaurus/docs/iOS/guides/go-live-checklist.md:58]: No, controllers are not observable objects, so they can't be state/observed objects in SwiftUI.

File: docusaurus/docs/iOS/guides/go-live-checklist.md (hunk 2 above)
Comment (martinmitrevski) [reply docusaurus/docs/iOS/guides/go-live-checklist.md]: Yes, I agree, updated

**Pull request merged**